tests/jballerina-unit-test/src/test/resources/test-src/record/open_record_type_inclusion_negative.bal

```diff
@@ -95,3 +95,17 @@ type Employee record {
     *PersonTwo;
     int age;
 };
+
+public type Foo record {
+    anydata body;
+};
+
+// Out of order inclusion test : added to test a NPE
+type Bar record {
+    *Foo;
+    Baz body;   // defined after the type definition
+};
+
+type Baz record {
+    int id;
+};
```

Comment:
These should work, right? I get a `redeclared symbol 'body'` error for this example.

But when I define `Baz` before `Bar` it compiles as expected.

```ballerina
public type Foo record {
    anydata body;
};

type Baz record {
    int id;
};

// Out of order inclusion test : added to test a NPE
type Bar record {
    *Foo;
    Baz body;   // defined after the type definition
};
```

Reply:
Yes, this should be allowed. I am looking into this. I have focused only on the crash.